### ISSUE TITLE
Write api docs

### ISSUE THREAD

**Issue by leahcornelius** [avrio-project/avrio-core#23]
## Description
We currently do not have a finished plan as to the format of the docs and hence we are making it up as we go along. We have this: https://github.com/avrio-project/avrio-docs but it needs finishing

If anyone has some time on their hands proposing some format would be great.

**Comment by leahcornelius** [comment]
Also if you implement a new API method ensure you write docs for it

**Comment by TheDevMinerTV** [comment]
Let's get a stable daemon first ^^

**Comment by leahcornelius** [comment]
The docs are written in gitbook, NOT using a raw editor. https://avrio-project.gitbook.io/avrio-core/

**Comment by leahcornelius** [comment]
Yes of course. However, im opening this so I can keep track of everything.

**Comment by leahcornelius** [comment]
Also once we have a decent API I can offload a lot into a separate wallet app which will make debugging the daemon a lot easier

**Comment by leahcornelius** [comment]
If you have any ideas (eg we should format the API as a rest API) then put them here. We will organise all API ideas in this
